pkg/resolver/config.go

```diff
@@ -9,23 +9,39 @@ import (
 
 // Config describes a [Resolver].
 type Config struct {
-	Name string `yaml:"name"`
-	Next string `yaml:"next,omitempty" toml:",omitempty" json:",omitempty"`
+	// Name is the unique name of this [Resolver]
+	Name string `yaml:""`
+	// Next is the name of the resolver to use if the Suffixes restriction
+	// isn't satisfied.
+	Next string `yaml:",omitempty" toml:",omitempty" json:",omitempty"`
 
 	// Debug indicates the requests passing through this [Resolver] should be logged or not.
-	Debug bool `yaml:"debug,omitempty"        toml:",omitempty" json:",omitempty"`
+	Debug bool `yaml:",omitempty" toml:",omitempty" json:",omitempty"`
 
 	// OmitSubNet indicates requests reaching out to remote servers should omit
 	// EDNS0 SUBNET information.
 	OmitSubNet bool `yaml:"omit_subnet,omitempty" toml:",omitempty" json:",omitempty"`
 
-	DisableAAAA bool     `yaml:"disable_aaaa,omitempty" toml:",omitempty" json:",omitempty"`
-	Iterative   bool     `yaml:"iterative,omitempty"    toml:",omitempty" json:",omitempty"`
-	Recursive   bool     `yaml:"recursive,omitempty"    toml:",omitempty" json:",omitempty"`
-	Servers     []string `yaml:"servers,omitempty"      toml:",omitempty" json:",omitempty"`
-	Suffixes    []string `yaml:"suffixes,omitempty"     toml:",omitempty" json:",omitempty"`
+	// DisableAAAA indicates this [Resolver] will discard AAAA entries
+	DisableAAAA bool `yaml:"disable_aaaa,omitempty" toml:",omitempty" json:",omitempty"`
 
-	Rewrites []RewriteConfig `yaml:"rewrite,omitempty" toml:",omitempty" json:",omitempty"`
+	// Iterative indicates this [Resolver] will go straight to the roots and
```

Comment:
Iterative indicates this *that* [Resolver] will go straight to the DNS root servers and

Reply:
> Iterative indicates this _that_ [Resolver] will go straight to the DNS root servers and

*that this*, sorry